src/ua.skrypnyk.hw10/Circle.java

```diff
@@ -0,0 +1,36 @@
+package src.ua.skrypnyk.hw10;
+
+import java.util.Scanner;
+
+public class Circle extends Shape {
+    private double perimeter;
+    private double area;
+
+    public Circle(double radius) {
+        this.radius = radius;
+    }
+
+    double radius = 11.1;
+    @Override
+    public double area() {
+        try {
+            if (radius <= 0) {
+                throw new isAreaOfShapeValid("Radius should be positive");
```

Comment:
Please, use more propriate name for the exception: 
For example: InvalidDataException